packages/base-controller/CHANGELOG.md

```diff
@@ -6,6 +6,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+### Added
+- Add `ControllerGetStateAction` and `ControllerStateChangeEvent` types ([#1890](https://github.com/MetaMask/core/pull/1890), [#2029](https://github.com/MetaMask/core/pull/2029))
+- Add `NamespacedName` type ([#1890](https://github.com/MetaMask/core/pull/1890))
+  - This is the narrowest supertype of all names defined within a given namespace.
+- Add `NotNamespacedBy` type, which matches a non-namespaced action/event name ([#2051](https://github.com/MetaMask/core/pull/2051))
```

Comment:
```suggestion
- Add `NotNamespacedBy` type, which matches an action/event name if and only if it is not prefixed by a given namespace ([#2051](https://github.com/MetaMask/core/pull/2051))
```
This is inaccurate, since the type does not return a positive result for non-namespaced names.

Reply:
That's not actually true, it returns a positive match for non-namespaced names. But that's more of a side effect. 

The entry should reflect the fact that the type is primarily intended to match namespaced action/event names that are prefixed by a *different* namespace than the one that is given.